Resources/config/default_settings.yml

```diff
@@ -0,0 +1,6 @@
+parameters:
+    # how many days banner should be hidden when user accepts policy?
+    ez_privacy_cookie.days: 365
+
+    # name to be used to store cookie status
+    ez_privacy_cookie.cookie_name: "EzPrivacyCookieStatus"
```

Comment:
I'm of the same opinion as @andrerom. Semantic configuration will require eZ Publish.